test/fails/test/__snapshots__/runner.test.ts.snap

```diff
@@ -36,7 +36,10 @@ TypeError: failure
 TypeError: failure"
 `;
 
-exports[`should fails > test-timeout.test.ts > test-timeout.test.ts 1`] = `"Error: Test timed out in 10ms."`;
+exports[`should fails > test-timeout.test.ts > test-timeout.test.ts 1`] = `
+"Error: Test timed out in 100ms.
+Error: Test timed out in 10ms."
```

Comment:
Shouldn't this have 3 errors?

Reply:
Maybe we should use another timer than. :)

Reply:
Ok, let me modify it.